.github/CHANGELOG.md

```diff
@@ -83,6 +57,245 @@
+      qml.RY(y, wires=1)
+      qml.CZ(wires=[1, 0])
+      qml.T(wires=0)
+      qml.SX(wires=0)
+      qml.Rot(x, y, z, wires=1)
+      qml.Rot(z, y, x, wires=1)
+      return qml.expval(qml.PauliX(wires=0))
+  ```
+
+  ```pycon
+  >>> optimized_circuit = qml.transforms.single_qubit_fusion()(circuit)
+  >>> dev = qml.device('default.qubit', wires=2)
+  >>> qnode = qml.QNode(optimized_circuit, dev)
+  >>> print(qml.draw(qnode)(0.1, 0.2, 0.3))
+   0: ──Rot(3.24, 1.57, 0)──╭Z──Rot(2.36, 1.57, -1.57)────┤ ⟨X⟩
+   1: ──Rot(3.14, 0.2, 0)───╰C──Rot(0.406, 0.382, 0.406)──┤
+  ```
+
+<h4>Improved Hamiltonian simulations</h4>
+
+* Added a sparse Hamiltonian observable and the functionality to support computing its expectation
+  value. [(#1398)](https://github.com/PennyLaneAI/pennylane/pull/1398)
+
+  For example, the following QNode returns the expectation value of a sparse Hamiltonian:
+
+  ```python
+  dev = qml.device("default.qubit", wires=2)
+
+  @qml.qnode(dev, diff_method="parameter-shift")
+  def circuit(param, H):
+      qml.PauliX(0)
+      qml.SingleExcitation(param, wires=[0, 1])
+      return qml.expval(qml.SparseHamiltonian(H, [0, 1]))
+  ```
+
+  We can execute this QNode, passing in a sparse identity matrix:
+
+  ```pycon
+  >>> print(circuit([0.5], scipy.sparse.eye(4).tocoo()))
+  0.9999999999999999
+  ```
+
+  The expectation value of the sparse Hamiltonian is computed directly, which leads to executions
+  that are faster by orders of magnitude. Note that "parameter-shift" is the only differentiation
+  method that is currently supported when the observable is a sparse Hamiltonian.
+
+* VQE problems can now be intuitively set up by passing the Hamiltonian
+  as an observable. [(#1474)](https://github.com/PennyLaneAI/pennylane/pull/1474)
+
+  ``` python
+  dev = qml.device("default.qubit", wires=2)
+  H = qml.Hamiltonian([1., 2., 3.],  [qml.PauliZ(0), qml.PauliY(0), qml.PauliZ(1)])
+  w = qml.init.strong_ent_layers_uniform(1, 2, seed=1967)
+
+  @qml.qnode(dev)
+  def circuit(w):
+      qml.templates.StronglyEntanglingLayers(w, wires=range(2))
+      return qml.expval(H)
+  ```
+
+  ```pycon
+  >>> print(circuit(w))
+  -1.5133943637878295
+  >>> print(qml.grad(circuit)(w))
+  [[[-8.32667268e-17  1.39122955e+00 -9.12462052e-02]
+  [ 1.02348685e-16 -7.77143238e-01 -1.74708049e-01]]]
+  ```
+
+  Note that other measurement types like `var(H)` or `sample(H)`, as well
+  as multiple expectations like `expval(H1), expval(H2)` are not supported.
+
+* Added functionality to compute the sparse matrix representation of a `qml.Hamiltonian` object.
+  [(#1394)](https://github.com/PennyLaneAI/pennylane/pull/1394)
+
+<h4> QNodes are more convenient</h4>
+
+* Added functionality to `qml.sample()` to extract samples from the basis states of
+  the device (currently only for qubit devices). Additionally, `wires` can be
+  specified to only return samples from those wires.
+  [(#1441)](https://github.com/PennyLaneAI/pennylane/pull/1441)
```

Comment:
```suggestion
* Computational basis samples directly from the underlying device can
  now be returned directly from QNodes via `qml.sample()`.
  [(#1441)](https://github.com/PennyLaneAI/pennylane/pull/1441)
```

Reply:
This reminds me: there are several demos we should update to use this :) E.g., the [QAOA demo](https://pennylane.ai/qml/demos/tutorial_qaoa_maxcut.html) comes to mind (I think there are others).

Currently, existing demos use two separate tricks to get the computational basis samples:

1. They use `qml.Hermitian`:

   ```python
   return qml.sample(qml.Hermitian(np.diag(2 ** np.arange(num_wires), wires=wires))

   res = qnode(params) # returns list of state samples in base-10
   output_samples = convert_to_binary(res) # need to convert to binary representation
   ```

2. Or, they use

   ```python
    return [qml.sample(qml.PauliZ(w)) for w in wires]

    res = qnode(params) # returns state samples using -1, 1
    output_samples = (res + 1) / 2 # convert to using 0, 1
    ```

Both of these approaches add overhead/complexity (_especially_ number 1!) and so should be updated. It also helps propagate to our users this new functionality.

Reply:
Made a note!